contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -78,6 +98,29 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     }
   }
 
+  /// @notice Updates a node operator
+  /// @param nodeOperatorIds The ID of the node operator being updated
+  function updateNodeOperators(uint256[] calldata nodeOperatorIds, NodeOperator[] calldata nodeOperators) external {
+    if (nodeOperatorIds.length != nodeOperators.length)
+      revert LengthMismatch(nodeOperatorIds.length, nodeOperators.length);
+    for (uint256 i; i < nodeOperatorIds.length; ++i) {
+      uint256 nodeOperatorId = nodeOperatorIds[i];
+      NodeOperator memory nodeOperator = nodeOperators[i];
+      if (nodeOperator.admin == address(0)) revert InvalidNodeOperatorAdmin();
+      if (msg.sender != nodeOperator.admin && msg.sender != owner()) revert AccessForbidden();
```

Comment:
Wait wouldn't switching it to "OR" always cause the transaction to fail?  The owner calling it will fail as it's not the `nodeOperator.admin` and the `nodeOperator.admin` is not the owner.  What do you mean by caching `owner()`?

Reply:
I was thinking
```
if (msg.sender = owner() || msg.sender = nodeOperator.admin) {
... do update
}
```

Reply:
I guess you could even merge this check if lines 108-110